src/RefractAPI.cc

```diff
@@ -38,6 +38,26 @@ namespace drafter {
             return ptr == NULL;
         }
 
+        void RemoveEmptyElements(RefractElements& elements)
+        {
+            elements.erase(std::remove_if(elements.begin(), elements.end(), IsNull<refract::IElement>), elements.end());
+        }
+
+        refract::IElement* DataStructureToRefractAST(const snowcrash::DataStructure& dataStructure)
```

Comment:
I want this to be named `DataStructureToRefract`. Please change the name of this and the other function.